docs/data/base/components/menu/MenuPopup.js

```diff
@@ -0,0 +1,169 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import MenuUnstyled from '@mui/base/MenuUnstyled';
+import MenuItemUnstyled, {
+  menuItemUnstyledClasses,
+} from '@mui/base/MenuItemUnstyled';
+import { styled } from '@mui/system';
+import { ClickAwayListener, PopperUnstyled } from '@mui/base';
+
+const blue = {
+  100: '#DAECFF',
+  200: '#99CCF3',
+  400: '#3399FF',
+  500: '#007FFF',
+  600: '#0072E5',
+  900: '#003A75',
+};
+
+const grey = {
+  100: '#E7EBF0',
+  200: '#E0E3E7',
+  300: '#CDD2D7',
+  400: '#B2BAC2',
+  500: '#A0AAB4',
+  600: '#6F7E8C',
+  700: '#3E5060',
+  800: '#2D3843',
+  900: '#1A2027',
+};
+
+const StyledMenu = styled(MenuUnstyled)(
+  ({ theme }) => `
+  font-family: IBM Plex Sans, sans-serif;
+  font-size: 0.875rem;
+  box-sizing: border-box;
+  padding: 5px;
+  margin: 10px 0;
+  min-width: 200px;
+  background: ${theme.palette.mode === 'dark' ? grey[900] : '#fff'};
+  border: 1px solid ${theme.palette.mode === 'dark' ? grey[800] : grey[300]};
+  border-radius: 0.75em;
+  color: ${theme.palette.mode === 'dark' ? grey[300] : grey[900]};
+  overflow: auto;
+  outline: 0px;
+  `,
+);
+
+const StyledMenuItem = styled(MenuItemUnstyled)(
+  ({ theme }) => `
+  list-style: none;
+  padding: 8px;
+  border-radius: 0.45em;
+  cursor: default;
+
+  &:last-of-type {
+    border-bottom: none;
+  }
+
+  &:focus-visible {
```

Comment:
Note that only the [technology preview of Safari supports this at the moment](https://caniuse.com/?search=%3Afocus-visible). We should recommend the polyfill in the meantime.

Generally, the demos should have the same support matrix the library has in my opinion.

Reply:
:+1: I forgot to implement the focus-visible polyfill for menu items. Will fix.